helper/testhelpers/minimal/minimal.go

```diff
@@ -4,6 +4,8 @@
 package minimal
 
 import (
+	"testing"
+
```

Comment:
This just needs the dreaded blank line removing and gofumpt etc to resort the imports.

Reply:
This is correct, actually -- there should be a line between golang imports and non-golang imports -- running `make fmt` makes no changes

Reply:
You're right I'm being a fool! Sorry. 😳